build/dockerfiles/base/Dockerfile

```diff
@@ -1,6 +1,6 @@
 FROM bitnami/git:latest AS git-from
 
-FROM --platform=$TARGETPLATFORM golang:1.19-bookworm
+FROM --platform=$TARGETPLATFORM golang:1.22.4-bookworm
```

Comment:
Is using `1.22` better?

Reply:
right, [1.22](https://github.com/docker-library/golang/blob/58bc678a838d467ca5afb7e0ee636cf5b8459da2/1.22/bookworm/Dockerfile) currently using 1.22.4, using `1.22` can automatically upgrade to a higher version without changing `Dockerfile`, done~